etl/naaccr_etl_postgresql.sql

```diff
@@ -853,10 +860,9 @@ CREATE TABLE naaccr_data_points_temp
     )
 
 
-    SELECT ( CASE WHEN  (SELECT MAX(measurement_id) FROM measurement) IS NULL
-        THEN 0
-        ELSE  (SELECT MAX(measurement_id) FROM measurement)
-      END + row_number() over (order by ndp.record_id)) AS measurement_id
+    SELECT COALESCE( (SELECT MAX(measurement_id) FROM measurement_temp)
```

Comment:
This change does not cause any problems but also does not fix any problem.  Since this is the first insert into MEASUMRENT_TEMP,  it is guaranteed to be empty.